packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js

```diff
@@ -74,7 +76,7 @@ function createWindow() {
           protocol: 'file:',
           slashes: true,
           webPreferences: {
-            devTools: false,
+            devTools: true,
```

Comment:
could set to false when in production

Reply:
yeah my bad this was debugging a test issue